internal/processors/hcgp/adapter.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Mia srl
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hcgp
+
+import (
+	"fmt"
+	"net/rpc"
+
+	"github.com/hashicorp/go-plugin"
```

Comment:
```suggestion
	"github.com/hashicorp/go-plugin"

```

Reply:
Are we dividing imports in this repo? I've seen mixed approaches on this matter and I though we weren't